.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@cosmocochrane1 can you remove you .vscode file?

Reply:
yup!